docs/Daos.md

```diff
@@ -0,0 +1,252 @@
+# Working with DAOs
+
+Arc.js provides a class called [DAO](api/classes/DAO) that facilitates creating and working with DAOs.  Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.
+
+## About DAOs
+
+Every DAO in DAOstack has an architecture designed by Arc that enables it to use the reusable contracts provided by Arc.  Each DAO has an avatar, a controller, a token, a reputation system, and a set of schemes and global constraints.
+
+**The avatar** is a public-facing part of the DAO that handles the interaction of the DAO with the rest of the world, for example interacting with other DAOs or paying third party agents.  A DAO is always referenced by the address of its avatar.
+
+**The controller** is a central internal hub for all of the components of the DAO.
+
+**The token** is used as currency in operations like staking in `GenesisProtocol`, vesting agreements in `VestingScheme` and rewards in `ContributionReward`.
+
+**The reputation system** provides a kind of currency of influence. Reputation conveys influence in the DAO commenserate with the rules 
+of the DAO, such as when voting and the nature of what causes it to be gained or lost.  Like tokens, reputation can be minted, but it cannot be transferred between agents.
```

Comment:
> ...but it cannot be transferred between agents.

what is agent ? if it is not defined here...so it is not clear (maybe just ...but it cannot be transferred to others).